app/serializers/provider_serializer.rb

```diff
@@ -103,4 +103,12 @@ def application_alert_recipient
       telephone: ''
       }]
   end
+
+  def generate_contacts
+    if object.ucas_preferences&.application_alert_email != nil
```

Comment:
Don't think we need the `!= nil` here. Can we check?

Reply:
You're right. Thanks Dan.